components/ReferenceManager/references/ReferencesContainer.tsx

```diff
@@ -152,6 +162,53 @@ function ReferencesContainer({
     ReactTooltip.rebuild();
   }, [selectedReferenceIDs]);
 
+  const { setReferenceTableRowData } = useReferencesTableContext();
+
+  const fetchCitationsWithQuery = async (url) => {
+    const config = api.GET_CONFIG();
+    config.headers["X-organization-id"] = currentOrg?.id.toString();
```

Comment:
In my branch I updated the different config such as `GET_CONFIG` to include optional headers. This is okay too for now.

Reply:
Ah nice. Will keep this for now and then we can make it more explicit in your branch